network/manager.go

```diff
@@ -337,9 +337,10 @@ func (nm *networkManager) CreateEndpoint(cli apipaClient, networkID string, epIn
 	}
 
 	if nw.VlanId != 0 {
-		if epInfo.Data[VlanIDKey] == nil {
+		// the first entry in epInfo is default address type
```

Comment:
update default address type to default nic in comment

Reply:
updated